mappings/net/minecraft/entity/passive/BeeEntity.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_4466 net/minecraft/entity/passive/BeeEntity
-	FIELD field_20353 multipleByteTracker Lnet/minecraft/class_2940;
-	FIELD field_20354 anger Lnet/minecraft/class_2940;
+	FIELD field_20353 MULTIPLE_BYTE_TRACKER Lnet/minecraft/class_2940;
```

Comment:
This is a really weird name... I think it should just be something like `STATUS_TRACKER`, rather than focusing on implementation details.